packages/react-native/Libraries/Text/Text/RCTTextView.mm

```diff
@@ -428,6 +444,98 @@ - (NSView *)hitTest:(NSPoint)point
   return isTextViewClick ? self : hitView;
 }
 
+- (NSNumber *)reactTagAtMouseLocationFromEvent:(NSEvent *)event
+{
+  NSPoint locationInSelf = [self convertPoint:event.locationInWindow fromView:nil];
+  NSPoint locationInInnerTextView = [self convertPoint:locationInSelf toView:_textView]; // This is needed if the parent <Text> view has padding
+  return [self reactTagAtPoint:locationInInnerTextView];
+}
+
+- (void)mouseEntered:(NSEvent *)event
+{
+  // superclass invokes self.onMouseEnter, so do this first
+  [super mouseEntered:event];
+
+  [self updateHoveredSubviewWithEvent:event];
+}
+
+- (void)mouseExited:(NSEvent *)event
+{
+  [self updateHoveredSubviewWithEvent:event];
+
+  // superclass invokes self.onMouseLeave, so do this last
+  [super mouseExited:event];
+}
+
+- (void)mouseMoved:(NSEvent *)event
+{
+  [self updateHoveredSubviewWithEvent:event];
+
+  // Web environments call mouse move events from the inside outwards, so do this last
```

Comment:
`[non-blocking|understanding]` what is this comment referring to?

Reply:
I ran a test using the simple web page below to see what kind of parity we would need to achieve. Even though we don't process mouse events right now, we might want to in the future, so I'm taking care of this detail now.

Perhaps I should try to find some more solid documentation on this?

```html
<html>
    <script>
        function callbacks(color) {
            const element = document.getElementById(color);
            element.addEventListener('mouseenter', (e) => console.log('enter ' + color + ', ' + JSON.stringify(e.clientX)));
            element.addEventListener('mouseleave', (e) => console.log('leave ' + color + ', ' + JSON.stringify(e.clientX)));
            //element.addEventListener('mousemove', (e) => console.log('move ' + color + ', ' + JSON.stringify(e.clientX)));
        }
        window.onload = () => {
            callbacks('gray');
            callbacks('yellow');
            callbacks('bold');
        }
    </script>
    <body>
        <span id="gray" style="background-color: silver;">
            Hello
            <span id="yellow" style="background-color: yellow;">
                <span id="bold" style="font-weight: bold;">
                    world!
                </span>
            </span>
        </span>
    </body>
</html>
```